python/ctsm/lilac_make_runtime_inputs.py

```diff
@@ -223,11 +187,11 @@ def buildnml(cime_path, rundir):
 
     # determine if fsurdat and/or finidat should appear in the -namelist option
     extra_namelist_opts = ''
-    if fsurdat != _UNSET:
-        # NOTE(wjs, 2020-06-30) With the current logic, fsurdat should never be _UNSET,
-        # but it's possible that this will change in the future.
+    if fsurdat != None:
```

Comment:
- [ ] Thanks for this change, but the pythonic way to check for None is: `if fsurdat is not None`. Can you please change this? (See https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_none.html)